main.html

```diff
@@ -0,0 +1,72 @@
+<html lang="en">
+
+<head>
+    <meta name="author" content="Jafar">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tugas menampilkan Post</title>
+
+    <style>
+        body {
+            margin: 0;
+            padding: 0;
+        }
+
+        h2 {
+            margin: 20px;
+            text-align: center;
+        }
+
+        .box {
+            margin: 30px;
+            padding: 20px auto;
+        }
+    </style>
+</head>
+
+<body>
+
+    <h2>Uncompleted tasks > 10</h2>
+    <div id='output' class="box"></div>
+    <script>
+        const main = async () => {
+            const user = await fetch('https://jsonplaceholder.typicode.com/users').then(response => response.json());
+            const todos = await fetch('https://jsonplaceholder.typicode.com/todos?completed=false').then(response => response.json());
+            const posts = await fetch("https://jsonplaceholder.typicode.com/posts").then(response => response.json());
```

Comment:
ketika handle asynchronous request (salah satunya pake `fetch`), jangan lupa error handling (dalam kasus ini, pake `catch()`)

Reply:
Okee kak terimakasih, aku bingung sih kak kalau pake axios harus pake server nodeJs juga ya kak?

Soalnya kalau engga, pas mau diambil datanya ke HTML pake document.getElementById() gitu, ada error message "UnhandledPromiseRejectionWarning: ReferenceError: document is not defined"

aku udah cari-cari juga tapi belum nemu wkwk